packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -465,12 +465,12 @@ const FormEdit = ({
 };
 
 export default function FormEditPage({
-  form,
   appUrl,
+  ...props
 }: inferSSRProps<typeof getServerSideProps> & { appUrl: string }) {
   return (
     <SingleForm
-      form={form}
+      {...props}
```

Comment:
Pass all props as is. It also has enrichedForm which wasn't being passed earlier,